lib/matplotlib/transforms.py

```diff
@@ -868,8 +867,8 @@ def update_from_path(self, path, ignore=None, updatex=True, updatey=True):
         if path.vertices.size == 0:
             return
 
-        points, minpos, changed = update_path_extents(
```

Comment:
From what I can tell, this was the only place that helper extension code is called. We should remove it here if it is dead code in a private module `_path`.

Reply:
Yeah, removed the python wrapper. The `update_path_extents` in `_path.h` is used one other place internal to that module, so left that alone.